core/src/main/resources/defaults/blueprints/7.2.11/cdp-data-engineering-ha.bp

```diff
@@ -494,15 +494,16 @@
           "hive-HIVESERVER2-BASE",
           "hms-GATEWAY-BASE",
           "hms-HIVEMETASTORE-BASE",
-          "oozie-OOZIE_SERVER-BASE",
```

Comment:
Probably related to Knox infront of the LB, and Basic supporting a single AS?

Reply:
The spirit of this change is, let's have a loadbalancable node group and put all the services that are needed to be behind the loadbalancer there because as you say basic supports only single AS. So Nginx, Knox and Oozie are in manager nodegroup.